Intersect.Server.Framework/Plugins/Helpers/IMapHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Intersect.Server.Framework.Items;
+using Intersect.Server.Framework.Maps;
+
+namespace Intersect.Server.Plugins.Helpers;
+
+public interface IMapHelper
```

Comment:
Add `public delegate void ItemAddedHandler(IItemSource?, IItem);`

The `?` is important since the source can be null and we want static analysis to not lie to us